app/app.go

```diff
@@ -451,18 +451,18 @@ func NewOsmosisApp(
 		app.GAMMKeeper,
 	)
 
-	app.LockupKeeper = *lockupKeeper.SetHooks(
-		lockuptypes.NewMultiLockupHooks(
-		// insert lockup hooks receivers here
-		),
-	)
-
 	app.IncentivesKeeper = *incentivesKeeper.SetHooks(
 		incentivestypes.NewMultiIncentiveHooks(
 		// insert incentive hooks receivers here
 		),
 	)
 
+	app.LockupKeeper = *lockupKeeper.SetHooks(
+		lockuptypes.NewMultiLockupHooks(
+			app.IncentivesKeeper.Hooks(),
+		),
+	)
```

Comment:
Any reason for moving this?

Reply:
LockupKeeper is now taking `IncentivesKeeper`, in which previously it didn't. 
I thought this should be done 'after' IncentivesKeeper has been declared, thus moved it below LockupKeeper.
Does the order of initialization not matter in this case?

Reply:
Ah this isn't built off main. I changed how this is all done in the keepers.go file there, so we no longer have to think about order here

Reply:
Cool, would have this back to the original state once this PR is ready and  merging with main